packages/core-components/src/lib/organisms/layout/EvidenceDefaultLayout.svelte

```diff
@@ -56,14 +58,17 @@
 		{logo}
 		{neverShowQueries}
 		{fullWidth}
+		{maxWidth}
+		{hideSidebar}
 		{githubRepo}
 		{slackCommunity}
 		{xProfile}
 		{algolia}
 	/>
 	<div
-		class={(fullWidth ? 'max-w-full ' : 'max-w-7xl ') +
+		class={(fullWidth ? 'max-w-full ' : maxWidth ? '' : ' max-w-7xl ') +
```

Comment:
is this the most natural way to introduce this prop?

I feel like maxWidth is mutually exclusive from fullWidth

Reply:
ignore this, that is the behaviour you have added